source/tutorial/ex3.rst

```diff
@@ -0,0 +1,59 @@
+.. _Exercise 3:
+
+Exercise 3 \\- \ SBL Shell/CLI
+---------------
+
+.. note::
+  **In this exercise, we'll learn how to execute SBL Shell Commands.**
```

Comment:
Please mention that the user can type `help<ENTER>` to list the available commands.

Reply:
fixed